### PR TITLE
Update SP guide

### PR DESCRIPTION
**Pull request by nikosev**

### CODE REVIEW

File: docs/developer/guide-for-sps.md [hunk 1]
```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
...
 
-This page contains information about connecting services to
-RCIAM in order to allow user
-login through RCIAM and to receive users' attributes. RCIAM is connected
-to a wide range of academic and social Identity Providers that users can choose
-from in order to access your service.
+This page contains information about connecting services to OpenAIRE Login in
+order to allow user login through OpenAIRE Login and to receive users'
+attributes. OpenAIRE Login is connected to a wide range of academic and social
+Identity Providers that users can choose from in order to access your service.
```
Comment (NicolasLiampotis) [inline docs/developer/guide-for-sps.md]: This page contains information about using the OpenAIRE AAI for securing access
to your service. The OpenAIRE AAI supports both OpenID Connect (an extension to
OAuth 2.0) and SAML 2.0. So the first thing you need to decide is which of the
two you are going to use. If you want you can also choose to secure some with
OpenID Connect and others with SAML. To secure your services you are also going
to need a library for the protocol you’ve selected. Once your
service is connected to the OpenAIRE AAI, users will be able to choose from
a wide range of academic and social Identity Providers in order to access your service.

File: docs/developer/guide-for-sps.md [hunk 2]
```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
...
 
-To integrate your Service Provider with the RCIAM service, you need to
-create a registration request using the
-[RCIAM Federation Registry Portal](https://federation-demo.rciam.grnet.gr). You
-can also use the Federation Registry portal to request the reconfiguration or
-deregistration of an existing deployed service. Service registration requests
-typically require approval by an administrator. Please refer to the
-[Federation Registry Documentation](https://federation.rciam.grnet.gr/docs) for
-more information.
+To integrate your Service Provider with the OpenAIRE Login service, you need to
```
Comment (NicolasLiampotis) [inline docs/developer/guide-for-sps.md]: "OpenAIRE Login service" --> "OpenAIRE AAI"

File: docs/developer/guide-for-sps.md [hunk 3]
```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
...
-   has identical functionality to the production instance, no information is
-   shared between the two systems.
-1. Register your Service Provider with the **production** instance of RCIAM
-   by selecting the "Production" integration environment during
-   registration through the RCIAM Federation Registry Portal. The production
-   instance allows access to your service through the academic and social
-   Identity Providers connected to RCIAM. This requires that your service
-   integration has been thoroughly tested during Step 1.
+1. Register your Service Provider and test integration with the **BETA**
+   instance of OpenAIRE Login. The BETA instance allows for testing
```
Comment (NicolasLiampotis) [inline docs/developer/guide-for-sps.md]: "OpenAIRE Login" --> "the OpenAIRE AAI"

File: docs/developer/guide-for-sps.md [hunk 4]
```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
...
+   authentication and authorisation without affecting the production OpenAIRE
+   Login service. Note that while the BETA instance has identical functionality
+   to the production instance, no information is shared between the two systems.
```
Comment (NicolasLiampotis) [inline docs/developer/guide-for-sps.md]: authentication and authorisation through the
available academic and social Identity Providers but shares no information
(e.g. users, groups or roles) with the production instance of the OpenAIRE AAI.

File: docs/developer/guide-for-sps.md [hunk 5]
```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
...
+1. Register your Service Provider with the **production** instance of OpenAIRE
+   Login to allow members of the OpenAIRE User Community to access your service.
+   This requires that the integration has been thoroughly tested during Step 1.
```
Comment (NicolasLiampotis) [inline docs/developer/guide-for-sps.md]: 1. Register your Service Provider with the **production** instance of the
OpenAIRE AAI. The production instance enables access to your service through any of the available  academic and social Identity Providers.
Moving to production requires that your service
integration has been thoroughly tested during Step 1.

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
OpenAIRE Login has been renamed as OpenAIRE AAI in the service catalogue so it's better to change "OpenAIRE Login service" to "OpenAIRE AAI"
See also comments in line

File: docs/developer/guide-for-sps.md (hunk 1 above)
Comment (nikosev) [reply docs/developer/guide-for-sps.md]: Done

File: docs/developer/guide-for-sps.md (hunk 2 above)
Comment (nikosev) [reply docs/developer/guide-for-sps.md]: Done

File: docs/developer/guide-for-sps.md (hunk 3 above)
Comment (nikosev) [reply docs/developer/guide-for-sps.md]: Done

File: docs/developer/guide-for-sps.md (hunk 4 above)
Comment (nikosev) [reply docs/developer/guide-for-sps.md]: Done

File: docs/developer/guide-for-sps.md (hunk 5 above)
Comment (nikosev) [reply docs/developer/guide-for-sps.md]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**